presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestingPrestoClient.java

```diff
@@ -85,6 +85,11 @@ public T execute(@Language("SQL") String sql)
 
     public T execute(Session session, @Language("SQL") String sql)
     {
+        return executeWithId(session, sql).getResult();
+    }
+
+    public ResultWithQueryId<T> executeWithId(Session session, @Language("SQL") String sql)
```

Comment:
Do you mean just this class or in the QueryRunner as well? Both are doable, but the latter touches ~20 files. I can push the incremental changes for that to a separate branch if you'd like to decide based on that.

Please let me know what way we go:
1. retrofit both TestingPrestoClient.execute and QueryRunner.execute
2. just the TestingPrestoClient.execute
3. leave as is